Tests/CommandsTests/PackageToolTests.swift

```diff
@@ -48,6 +49,22 @@ final class PackageToolTests: XCTestCase {
         XCTAssert(stdout.contains("Swift Package Manager"), "got stdout:\n" + stdout)
     }
     
+    func testInitHelp() throws {
+        AssertHelp(for: Commands.SwiftPackageTool.Init.self, equals: """
+            OVERVIEW: Initialize a new package
+
+            USAGE: init [<options>] --enable-index-store
```

Comment:
--enable-index-store ?

Reply:
` but it does go away when they're all transitioned.` what does that mean?

Reply:
another point is that I dont think its scalable to string match the the entire output of the `--help` for each command. did you intend to add such tests to all commands or just this one as an indicator fo the rest?

Reply:
It doesn't have to be for all the commands, I think it would be enough to have tests for the most popular commands: e.g. `swift run`, `swift build`, `swift test` 

`--enable-index-store` goes away once all the commands have the option group hidden. In my initial testing that's when I noticed that it would go away.